entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+openapi_python_client_version=$1
+openapi_file_path=$2
+config_file_path=$3
+
+if [[ "$openapi_python_client_version" != "NOT_SPECIFIED" ]]; then
+    version_arg="--spec openapi-python-client==${openapi_python_client_version}"
+else
+    version_arg=""
+fi
+
+if [[ "$config_file_path" != "NOT_SPECIFIED" ]]; then
+    config_arg="--config ${config_file_path}"
+else
+    config_arg=""
+fi
+
+pipx run ${version_arg} openapi-python-client ${config_arg} generate --path ${openapi_file_path}
```

Comment:
Might be better to `pipx install` this in the Dockerfile, then reuse the same version? Seems unlikely that someone would generate multiple clients in a single CI pipeline but we've done weirder things.

Reply:
AFAIK theres no way to use inputs inside the Dockerfile unfortunately :(